packages/workbox-precaching/utils/printInstallDetails.mjs

```diff
@@ -7,57 +7,53 @@
 */
 
 import {logger} from 'workbox-core/_private/logger.mjs';
+
 import '../_version.mjs';
 
 /**
  * @param {string} groupTitle
- * @param {Array<PrecacheEntry>} entries
+ * @param {Array<string>} urls
  *
  * @private
  */
-const _nestedGroup = (groupTitle, entries) => {
-  if (entries.length === 0) {
+function _nestedGroup(groupTitle, urls) {
+  if (urls.length === 0) {
     return;
   }
 
   logger.groupCollapsed(groupTitle);
 
-  entries.forEach((entry) => {
-    logger.log(entry._originalInput);
-  });
+  for (const url of urls) {
+    logger.log(url);
+  }
 
   logger.groupEnd();
-};
+}
 
 /**
- * @param {Array<Object>} entriesToPrecache
- * @param {Array<Object>} alreadyPrecachedEntries
+ * @param {Array<string>} urlsToPrecache
+ * @param {Array<string>} urlsAlreadyPrecached
  *
  * @private
- * @memberof module:workbox-precachig
+ * @memberof module:workbox-precaching
  */
-export default (entriesToPrecache, alreadyPrecachedEntries) => {
-  // Goal is to print the message:
-  //    Precaching X files.
-  // Or:
-  //    Precaching X files. Y files were cached and up-to-date.
-
-  const precachedCount = entriesToPrecache.length;
-  const alreadyPrecachedCount = alreadyPrecachedEntries.length;
-  let printText =
-    `Precaching ${precachedCount} file${precachedCount === 1 ? '' : 's'}.`;
-  if (alreadyPrecachedCount > 0) {
-    printText += ` ${alreadyPrecachedCount} ` +
-      `file${alreadyPrecachedCount === 1 ? ' is' : 's are'} already cached.`;
-  }
+export default (urlsToPrecache, urlsAlreadyPrecached) => {
+  const precachedCount = urlsToPrecache.length;
+  const alreadyPrecachedCount = urlsAlreadyPrecached.length;
 
-  logger.groupCollapsed(printText);
+  if (precachedCount || alreadyPrecachedCount) {
+    let message =
+        `Precaching ${precachedCount} file${precachedCount === 1 ? '' : 's'}.`;
 
-  _nestedGroup(
-      `View precached URLs.`,
-      entriesToPrecache);
-  _nestedGroup(
-      `View URLs that were already precached.`,
-      alreadyPrecachedEntries);
-  logger.groupEnd();
+    if (alreadyPrecachedCount > 0) {
+      message += ` ${alreadyPrecachedCount} ` +
+        `file${alreadyPrecachedCount === 1 ? ' is' : 's are'} already cached.`;
+    }
+
+    logger.groupCollapsed(message);
```

Comment:
Should this all be inside a `NODE_ENV !== 'production'` conditional?

Reply:
The code that calls this module is inside a `NODE_ENV` check.

Reply:
But uglify may not be able to figure that out. Can you confirm that it doesn't appear in the minified file?

Reply:
http://storage.googleapis.com/workbox-cdn/releases/4.0.0-beta.0/workbox-precaching.prod.js suggests that it's not in the `prod` bundle, due to what I guess is tree-shaking.

This PR changes some of the logging logic in this file, but shouldn't lead to changes in how things are bundled up.